README.md

```diff
@@ -50,6 +50,12 @@ There are methods for the four canonical event types: `created`, `updated`,
 the publisher floods with `noop`s for all existing entities so subscribers can
 refresh their view of the domain.
 
+A timestamp argument may be passed (it will be set by the bus automatically
+otherwise); it must be an integer number of milliseconds since the UNIX Epoch:
+
+```ruby
+client.created('widgets', 'https://app.example.com/widgets/1', 1473080555409)
```

Comment:
I suppose this was added in another pull-request? I'm wondering whether that could cause issues when debugging. For instance, if any arbitrary time can be passed in the message what would be a good source of truth when looking at time-sensitive events?